inlineplz/main.py

```diff
@@ -21,14 +20,18 @@
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--pull-request', type=int)
+    parser.add_argument('--pull-request', type=int, default=None)
     parser.add_argument('--owner', type=str)
     parser.add_argument('--repo', type=str)
     parser.add_argument('--repo-slug', type=str)
-    parser.add_argument('--branch', type=str)
-    parser.add_argument('--token', type=str)
+    parser.add_argument('--branch', type=str, default=None)
+    parser.add_argument('--token', type=str, default=None)
     parser.add_argument('--interface', type=str, choices=interfaces.INTERFACES)
-    parser.add_argument('--url', type=str)
+    parser.add_argument('--url', type=str, default=None)
+    parser.add_argument('--username', type=str)
+    parser.add_argument('--password', type=str)
+    parser.add_argument('--host', type=str)
+    parser.add_argument('--topic', type=str)
```

Comment:
could we rename this to something like `review-id`? and deprecate `pull-request`?

Reply:
Yes, we could do review-id, I like that.